Example from README: mirage: unknown option --unix

After installing mirage 2.7.3 from OPAM on a Debian X86, examples given in the README result in:

    :mirage-skeleton $ make sample-configure
    mirage configure -f sample/config.ml --unix 
    mirage: unknown option `--unix'.
    Usage: mirage configure [OPTION]... 
    Try `mirage configure --help' or `mirage --help' for more information.
    Makefile:41: recipe for target 'sample-configure' failed
    make: *** [sample-configure] Error 1

I also noticed that there is no file `sample/config.ml` file. Maybe sample refers to `hello` and so on but this isn't obvious. At this point I have no idea how to go on.

--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,6 @@ endif
 
 CONFIGS = $(patsubst %, %-configure, $(TESTS))
 BUILDS  = $(patsubst %, %-build,     $(TESTS))
-RUNS    = $(patsubst %, %-run,       $(TESTS))
 TESTRUN = $(patsubst %, %-testrun,   $(TESTS))
 CLEANS  = $(patsubst %, %-clean,     $(TESTS))
 
@@ -18,7 +17,6 @@ all: build
 
 configure: $(CONFIGS)
 build: $(BUILDS) lwt-build
-run: $(RUNS)
 testrun: $(TESTRUN)
 clean: $(CLEANS) lwt-clean
 

--- a/README.md
+++ b/README.md
@@ -28,18 +28,26 @@ Configure, Build, Run
 
 Each example is invoked in the same way:
 
-    $ make sample-configure
-    $ make sample-build
-    $ make sample-run
+    $ make ${example}-configure
+    $ make ${example}-build
+
+So to configure the hello example, run `make hello-configure`, and to build it, run `make hello-build`.
+The binaries built in the process can then be found in the example's directory:
+
+    $ cd hello/
+    $ ./mir-console
+    Hello World!
+    Hello World!
+    ...
 
 If you want to clean up afterwards, the usual does the trick:
 
-    $ make sample-clean
+    $ make ${example}-clean
 
 Some global targets are also provided in the `Makefile`:
 
     $ make all                   ## equivalent to ...
-    $ make configure build run
+    $ make configure build
     $ make clean
 
 Details

